app/services/import_service.rb

```diff
@@ -101,9 +104,13 @@ def create_folder(compressed_file_name)
     folder
   end
 
-  def remove_file_references(user)
-    user.remove_exported_photos_file
-    user.remove_export
-    user.save
+  # Removes import files after processing
+  # @param [*String] files
+  def remove_import_files(*files)
+    files.each do |file|
+      if file && File.exist?(file)
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.